contrib/babelfishpg_tsql/sql/sys_procedures.sql

```diff
@@ -165,9 +165,9 @@ GRANT EXECUTE ON PROCEDURE sys.sp_babelfish_configure(
 	IN varchar(128), IN varchar(128), IN varchar(128)
 ) TO PUBLIC;
 
-CREATE OR REPLACE PROCEDURE sys.sp_addrole(IN "@rolname" sys.SYSNAME)
+CREATE OR REPLACE PROCEDURE sys.sp_addrole(IN "@rolname" sys.SYSNAME, IN sys.SYSNAME DEFAULT NULL)
```

Comment:
We shouldn't be doing this change because right now we cannot deal with the second argument - we need further investigation to check if the meaning of "role owner" is the same in SQL server and in PG